blocks/extensions/monitoring.py

```diff
@@ -41,23 +41,24 @@ def add_records(self, log, record_tuples):
 
 
 class DataStreamMonitoring(SimpleExtension, MonitoringExtension):
-    """Monitors values of Theano variables on a data stream.
+    """Monitors values of Theano and non-Theano variables on a data stream.
```

Comment:
I saw it before but did not mention because there were important issues, but now I think it's the time to decide on terminology. I am not a fan of `non-Theano variables` because those are not variables. If we call them monitored _quantities_, please make sure that this term is used throughout the documentation.